_includes/sidebar-releases.json

```diff
@@ -55,36 +55,7 @@
             "/releases/v20.2.3.html",
             "/releases/v20.2.2.html",
             "/releases/v20.2.1.html",
-            "/releases/v20.2.0.html",
```

Comment:
LGTM! Just curious, why all these deletions?

Reply:
I figured we don't need these represented in the main nav anymore since they are past (unsupported) releases